app/src/main/res/values/strings.xml

```diff
@@ -7,6 +7,15 @@
     <!-- App name for private browsing mode. The first parameter is the name of the app defined in app_name (for example: Fenix)-->
     <string name="app_name_private_4">%s (Private)</string>
 
+    <!-- Labels common to multiple components -->
+    <string name="common_label_close">Close</string>
+    <string name="common_label_delete">Delete</string>
+    <string name="common_label_edit">Edit</string>
+    <string name="common_label_off">Off</string>
+    <string name="common_label_on">On</string>
+    <string name="common_label_remove">Remove</string>
+    <string name="common_label_save">Save</string>
```

Comment:
These labels would probably need individual comments to help localizers understand the context of where they are used.
For example this is how each string (in this case `recently_saved_menu_item_remove`) would appear for localizers - https://pontoon.mozilla.org/ro/firefox-for-android/mozilla-mobile/fenix/app/src/main/res/values/strings.xml/?search=recently_saved_menu_item_remove&string=242751